README.md

```diff
@@ -27,9 +27,14 @@ _If you collaborate in this repo, please always sort the gist in *Alphabetical O
 
 ## Gradle & Compile
 
-[Speddup debug compile time] (https://gist.github.com/Aracem/a2bcdb841d9d2df07c23)
+[Speed up debug compile time] (https://gist.github.com/Aracem/a2bcdb841d9d2df07c23)
+
 * Speed up the compile time of your debug buildtype. Add this manifest to your debug folder and set API target to 22
```

Comment:
Well, in my opinion is a gist to how add a special Manifest with the vmSafeMode. The wiki is a complete guide with all the possible tips

Anyway, :+1:  to this pull request